content/cloud/kv-cloud-tutorial.md

```diff
@@ -0,0 +1,130 @@
+
+This tutorial will focus primarily on linking your Spin applications to Key Value Store objects on Fermyon Cloud. To learn more how to configure business login within a Spin application to access key value data, please visit the [Spin Key Value tutorial](../spin/v2/key-value-store-tutorial). If you aren't familiar with the concept of linking your Spin application to a Fermyon Cloud storage object, the short summary is linking connects your Spin application to a Fermyon Cloud managed store via an associated label. That label is scoped to the Spin application you're linking and only that Spin application. This creates a seperation of concerns for application developers who now do not need to worry about the naming specifics of the data store, only that their application should have access. For an indepth review of these concepts, please visit [the links and labels conceptual article](linking-applications-to-resources-using-labels.md)
+
+> Please check that you have [Spin CLI](./cli-reference.md) v2.0 or greater, and [`cloud` plugin](https://github.com/fermyon/cloud-plugin) v0.7.0 or greater installed. 
+
+
+## Creating a New Spin Application
+
+If you already have a Spin application, you can skip this step. If you do not have a Spin application locally you will need to [create one](/spin/quickstart#create-your-first-application).
+
+## Grant Key Value Store Permission
+
+To tell Spin that we want to use a Key Value Store, we only need to [grant Key Value Store permission to a component in the application’s manifest](/spin/kv-store-api-guide#granting-key-value-store-permissions-to-components) (the `spin.toml` file) by supplying a [label](linking-applications-to-resources-using-labels.md). For example:
+
+```
+[component.example]
+key_value_stores = ["default"]
+```
+
+The new database file (`sqlite_key_value.db`) is created in your application's `.spin` folder when you run this Spin application. The database persists across Spin application invocations and updates. The key value store is empty to start.
+
+## Creating A Key Value Store in Fermyon Cloud
+
+You have two options to create your key value store (both are equally valid):
+* Option A: Implicitly create a key value store (with an automatically assigned arbitrary database name) via `spin cloud deploy`. For example `inspirational-pig` or similarly named database will be created.
+* Option B : Explicitly provide the name of the database to create via the `spin cloud kv create` command. For example, `spin cloud kv create finance-database` creates a database called `finance-database` inside your Fermyon Cloud account.
+
+Please run whatever step is relevant to your needs.
+
+### Implicitly Create A Key Value Store Via `spin cloud deploy`
```

Comment:
```suggestion
<!-- markdownlint-disable-next-line titlecase-rule -->
### Implicitly Create A Key Value Store Via `spin cloud deploy`
```

Reply:
Allowing build/test to ignore Title Case in situations like this where we use the command as part of the heading.